### PR TITLE
feat(ui): setup mobx, add home page, main page after login with sidebar

### PR DESCRIPTION
**Pull request by gordonpn**

### CODE REVIEW

**Comment by Mackbellemore** [comment]
Task linked: [CU-kh1ner Home page and Main Page with sidebar](https://app.clickup.com/t/kh1ner)

**Comment by Mackbellemore** [comment]
Task linked: [CU-kuyp56 Implement MobX state-management](https://app.clickup.com/t/kuyp56)

File: Makefile [hunk 1]
```diff
@@ -14,18 +14,18 @@ help: ## Show this help
 	@egrep -h '\s##\s' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[33m%-20s\033[0m %s\n", $$1, $$2}'
 
 run: ## Start up both client and server containers
-	$(CMD_DOCKER_BUILD) --force-recreate
+	$(CMD_DOCKER_BUILD)
```
Comment (nasjuice) [inline Makefile:17]: make up your mind

Comment (gordonpn) [reply Makefile:17]: I didn't want it anymore

File: client/src/common/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import React, { useContext } from 'react';
+import HamburgerMenu from 'react-hamburger-menu';
+import { RootStoreContext } from '../stores/stores';
+
+export const styles = {
+  bmMenuWrap: {
+    position: 'fixed',
+    height: '100%',
+  },
+  bmMenu: {
```
Comment (nasjuice) [inline client/src/common/Sidebar.jsx:11]: where are we using these

Comment (gordonpn) [reply client/src/common/Sidebar.jsx:11]: They are CSS classes specific for the hamburger menu react library.

**Review by nasjuice — APPROVED** [review]

File: client/src/utils/api.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+export const makeGetRequest = async (slug) => {
+  const apiUrl = process.env.NEXT_PUBLIC_API_URL;
+  const { data } = await axios.get(`${apiUrl}/${slug}`);
```
Comment (Mackbellemore) [inline client/src/utils/api.js:5]: I believe you need a {withCredentials: true} here

Comment (gordonpn) [reply client/src/utils/api.js:5]: That'll come with the refactoring right after this sprint ;)

That function isn't used yet

Comment (Mackbellemore) [reply client/src/utils/api.js:5]: ahh thought it was ayyyyt g

**Review by Mackbellemore — APPROVED** [review]
:garlic:

**Pull request merged**